README.md

```diff
@@ -43,14 +43,13 @@ c := polygon.NewWithClient("YOUR_API_KEY", hc)
 After creating the client, making calls to the Polygon API is simple.
 
 ```golang
-params := models.GetAllTickersSnapshotParams{
-    Locale:     models.US,
-    MarketType: models.Stocks,
-}.WithTickers("AAPL,MSFT")
+params := models.GetTickerDetailsParams{
+Ticker: "AAPL",
+}
```

Comment:
can we add `WithDate(...)` to the example so that we're still demonstrating how to use query params?

Reply:
ty